src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/LogRecordExtensions.cs

```diff
@@ -33,29 +33,73 @@ internal static class LogRecordExtensions
     {
         private static readonly string[] SeverityTextMapping = new string[]
         {
-            null, "Trace", "Debug", "Information", "Warning", "Error", "Fatal",
+            "Trace", "Debug", "Information", "Warning", "Error", "Fatal",
```

Comment:
Something to come back to in another PR maybe...

I think these are somewhat inspired by a combination of ILogger and the LogRecord data model... but technically, I think SeverityText is supposed to be the name as it's known by the logging framework being used see: https://github.com/open-telemetry/opentelemetry-proto/blob/724e427879e3d2bae2edc0218fff06e37b9eb46e/opentelemetry/proto/logs/v1/logs.proto#L144-L146

The SeverityNumber (which I don't think we're using today) on the other hand is governed by the data model. But even then, each logging framework should probably be responsible for defining any mapping from its severity levels to one of the normalized OTLP severity numbers. See https://github.com/open-telemetry/opentelemetry-proto/blob/724e427879e3d2bae2edc0218fff06e37b9eb46e/opentelemetry/proto/logs/v1/logs.proto#L140-L142

Reply:
Wow interesting! At the moment on this branch, you have to specify a spec value or `null`: https://github.com/open-telemetry/opentelemetry-dotnet/blob/741966832a814fb3a60c163a3536192289c22f26/src/OpenTelemetry.Api/Logs/LogRecordData.cs#L95

That makes it the appender's job to map to spec a la: https://github.com/open-telemetry/opentelemetry-dotnet/blob/741966832a814fb3a60c163a3536192289c22f26/src/OpenTelemetry.Extensions.EventSource/OpenTelemetryEventSourceLogEmitter.cs#L209-L219

We'll have to make changes to support passing the raw original log level as a string in addition to the spec value. Is the raw non-normalized value useful? 🤣

Reply:
> Is the raw non-normalized value useful?

Yea, hard to say, maybe?

If you've been using some logging framework for 30 years, you're probably pretty accustomed to its nomenclature and not so much OpenTelemetry's. So, I guess it might make analyzing logs more familiar to an end user like this?

Also see https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#field-severitytext